avaframe/com1DFA/deriveParameterSet.py

```diff
@@ -149,6 +149,7 @@ def checkResType(fullCfg, section, key, value):
             "TA",
             "particles",
             "dmDet",
+            "FTDet"
```

Comment:
Add to output info https://docs.avaframe.org/en/latest/moduleCom1DFA.html#output and to com1Cfg.ini info (on resType parameter)

Reply:
added int the outputs